### ISSUE TITLE
Please support Redmi K30 Pro/POCO F2 Pro

### ISSUE THREAD

**Issue by droople** [sriharshaarangi/BatteryChargeLimit#194]
Hi There, 
It's not working on Redmi K30 Pro/POCO F2 Pro.
The control file location is  "/sys/class/power_supply/battery/battery_charging_enabled", could you please add the support?

Regards

**Comment by jason-eu** [comment]
Any update?

**Comment by milux** [comment]
No, but you can set the control file manually in settings. (And also verify that it actually works, as a bonus.)

**Comment by jason-eu** [comment]
> No, but you can set the control file manually in settings. (And also verify that it actually works, as a bonus.)

Thank you!
